src/makeCalculator.js

```diff
@@ -34,10 +34,46 @@
  * calculator.result === -28
  *
  *
- * @return {object}
+
  */
+
 function makeCalculator() {
-  // write code here
+  const obj = {
+    add(a) {
+      obj.result += a;
+
+      return obj.result;
+    },
+    subtract(a) {
+      obj.result -= a;
+
+      return obj.result;
+    },
+    multiply(a) {
```

Comment:
```suggestion
    subtract(a) {
      obj.result -= a;

      return obj.result;
    },
    
    multiply(a) {
```

Next time pls add blank line between methods